usecases/traverser/explorer.go

```diff
@@ -268,7 +283,26 @@ func (e *Explorer) Concepts(ctx context.Context,
 	return results, nil
 }
 
-func (e *Explorer) vectorFromExploreParams(ctx context.Context,
+func (e *Explorer) vectorFromParams(ctx context.Context,
+	params GetParams) ([]float32, error) {
+	if params.NearText != nil && params.NearVector != nil {
+		return nil, errors.Errorf("found both 'nearText' and 'nearVector' parameters " +
+			"which are conflicting, choose one instead")
+	}
+
+	if params.NearText != nil {
+		vector, err := e.vectorFromNearTextParams(ctx, params.NearText)
+		if err != nil {
+			return nil, fmt.Errorf("vectorize params: %v", err)
+		}
+
+		return vector, nil
+	}
+
+	return params.NearVector.Vector, nil
```

Comment:
I think it can never be nil, because `vectorFromParams` is only ever called from `getClassExploration` which in turn is only called if one of the `near<>` is not nil. So since `NearText` is nil, `NearVector` has to be not-nil, otherwise we wouldn't even land here.

https://github.com/semi-technologies/weaviate/blob/eb500dea9f826c23aa52e4e563a821b387664888/usecases/traverser/explorer.go#L77-L84